### PR TITLE
adds support for multiple backing token types in Gnosis Safe + lint fixes

### PR DESCRIPTION
**Pull request by holtzman**
The mainnet gnosis safe now backs eUSD with both RSV and eUSD.

This is a quick and not-quite complete fix for the front end to recognize both.

The Gnosis Safe Summary block shows both token balances, as well as rolling them up to a total balance

Todo:

Deposits and Withdrawals are showing both RSV and eUSD withdrawals but need a symbol lookup as all are identified using the single symbol being stored in the config file.  We could fix this either by looking up the contract address -> symbol mapping from the gnosis safe, available at least in the getBalances call, or by extending our config file to have that map with it.

We would like to merge this PR with only any absolutely necessary suggested updates and then make improvements, as the production reserve-auditor really needs these fixes as a hotfix.

### CODE REVIEW

File: frontend/src/App.tsx [hunk 1]
```diff
@@ -6,10 +6,10 @@ import { AuditList } from './layout/AuditList'
...
 import { GnosisSafeProvider } from './contexts'
 
-let bg_color = "#004e80" // mainnet background color is a shade of blue
-if (typeof MC_NETWORK  !== 'undefined') {
+let bg_color = '#004e80' // mainnet background color is a shade of blue
+if (typeof MC_NETWORK !== 'undefined') {
   if (MC_NETWORK == 'testnet') {
-    bg_color = "#774e80" // testnet background color is lavender
+    bg_color = '#774e80' // testnet background color is lavender
   }
```
Comment (holtzman) [inline frontend/src/App.tsx:13]: These changes were made by the linter (yarn lint)

File: frontend/src/App.tsx [hunk 2]
```diff
@@ -42,7 +42,7 @@ export const App = () => {
             height: '100vh',
             display: 'flex',
             flexDirection: 'column',
-            backgroundColor: `${bg_color}`
+            backgroundColor: `${bg_color}`,
           }}
```
Comment (holtzman) [inline frontend/src/App.tsx:46]: more linting

File: frontend/src/components/GnosisSafe.tsx [hunk 3]
```diff
@@ -2,12 +2,11 @@ import { Box, Link, Tooltip, Typography } from '@mui/material'
...
 import { GnosisSafeContext } from '../contexts'
-import { TGnosisSafeUsdBalanceResponse } from '../types'
 
-let gnosis_net = 'eth'; // default to a safe on the eth main network
+let gnosis_net = 'eth' // default to a safe on the eth main network
 if (typeof MC_NETWORK !== 'undefined') {
   if (MC_NETWORK == 'testnet') {
-    gnosis_net = 'gor'; // the testnet bridge uses the goerli eth test network
+    gnosis_net = 'gor' // the testnet bridge uses the goerli eth test network
   }
```
Comment (holtzman) [inline frontend/src/components/GnosisSafe.tsx:10]: more linter changes

File: frontend/src/components/Withdrawal.tsx [hunk 4]
```diff
@@ -71,7 +71,7 @@ export const Withdrawal: FC<TProps> = ({ withdrawal }: TProps) => {
         </Tooltip>
       </Box>
     </Box>
-  )  : (
+  ) : (
```
Comment (holtzman) [inline frontend/src/components/Withdrawal.tsx:74]: linter changes

File: frontend/src/layout/AuditList.tsx [hunk 5]
```diff
@@ -120,12 +120,9 @@ export const AuditList: FC = (): ReactElement => {
...
                   <AuditedBurn auditedBurn={i} key={index} />
                 ))}
-
               </InfiniteScroll>
-            }
-            {burns.length == 0 &&
-              <h4>&nbsp;&nbsp;None</h4>
-            }
+            )}
+            {burns.length == 0 && <h4>&nbsp;&nbsp;None</h4>}
```
Comment (holtzman) [inline frontend/src/layout/AuditList.tsx:125]: linter at work

File: frontend/src/layout/Header.tsx [hunk 6]
```diff
@@ -2,8 +2,9 @@ import React, { FC, ReactElement } from 'react'
 import { Box, Container, Toolbar, Typography } from '@mui/material'
 import { eUSDIcon } from '../components/icons'
 
-let net_name = ""; // network name prefix for the header, blank for mainnet.
-if (typeof MC_NETWORK !== 'undefined') {  // env variable set in webpack.<net>.js.
+let net_name = '' // network name prefix for the header, blank for mainnet.
+if (typeof MC_NETWORK !== 'undefined') {
+  // env variable set in webpack.<net>.js.
   if (MC_NETWORK == 'testnet') {
```
Comment (holtzman) [inline frontend/src/layout/Header.tsx:8]: linter

File: frontend/src/types/api.ts [hunk 7]
```diff
@@ -215,4 +215,4 @@ export type TGnosisSafeAllTransactionsListResponse = {
     | SafeMultisigTransactionWithTransfersResponse
     | EthereumTxWithTransfersResponse
   >
-}
\ No newline at end of file
+}
```
Comment (holtzman) [inline frontend/src/types/api.ts:218]: linter

**Review by holtzman — COMMENTED** [review]
pointed out where linter made changes.

**Review by david1alvarez — APPROVED** [review]

**Pull request merged**